### DISCUSSION TITLE
Why does Kong consume so much memory?

### DISCUSSION THREAD

**Discussion by masksshow** [Kong/kong#8165] [Help]
I read the documentation, the cong is based on Nginx. 
I set nginx_worker_processes = 1
My Nginx container consumes 3 mb. Kong consumes 115mb
What is wrong with it?

**Comment by fffonion** [comment]
Please refer to [sizing guidelines](https://docs.konghq.com/gateway/2.6.x/plan-and-deploy/sizing-guidelines/) and metrics
tools like [prometheus plugin](https://docs.konghq.com/hub/kong-inc/prometheus/) to find out what takes the memory of Kong.

**Reply by masksshow** [reply]
I have read all the documentation. It doesn't say a word about how much memory Kong consumes.
I have changed all configurations to the minimum. But it had no effect.
Why kong uses more than 100mb. Is this the norm?
